src/main/java/CodeCatalyst.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Scanner;
+
+public class CodeCatalyst {
+    private static final int MAX_TASKS = 100;
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        Task[] tasks = new Task[MAX_TASKS];
+        int taskCount = 0;
+
```

Comment:
The set up of scanner, tasks, and taskCount could be refactored out into a separate set up function.